apps/staking/components/DropdownHamburgerMenu.tsx

```diff
@@ -61,6 +64,13 @@ export function DropdownHamburgerMenu() {
         {EXTERNAL_ROUTES.map(({ dictionaryKey, href }) => (
           <DropdownMenuItemNavLink key={href} href={href} label={navDictionary(dictionaryKey)} />
         ))}
+        <DropdownMenuItemNavLink
+          href={URL.TOKEN_UNISWAP_POOL_ARB}
+          className={cn('inline-flex items-center gap-1.5', 'md:hidden')}
+        >
+          <SessionTokenIcon className={cn('h-4 w-4')} />
```

Comment:
nit: remove uses of `cn` function and curly brackets where not needed, first usage is arguably good because it separates breakpoints, but the Icon usage is not need